core/include/mm/vm.h

```diff
@@ -87,26 +83,20 @@ TEE_Result vm_buf_to_mboj_offs(const struct user_mode_ctx *uctx,
 			       const void *va, size_t size,
 			       struct mobj **mobj, size_t *offs);
 
-/*-----------------------------------------------------------------------------
- * vm_va2pa - Translate virtual user address to physical address
- * given the user context.
- * Interface is deprecated, use virt_to_phys() instead.
- *---------------------------------------------------------------------------*/
+/* This function is far less efficient than virt_to_phys() */
```

Comment:
Instead of this comment it could be worth noting that this is a helper function for `virt_to_phys()` and shouldn't be used directly elsewhere.